Showcase-iOS/Login/View/LoginVC.swift

```diff
@@ -10,8 +10,29 @@ import UIKit
 
 class LoginVC: UIViewController {
 
+    var loginPresenter: LoginPresentable?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
     }
 
 }
+
+extension LoginVC: LoginPresenterViewable {
```

Comment:
Removed this until you use it, doesnt make any sense to have committed empty methods. Add this back when you use it. This PR says that you are creating a Dependency Container but now you are adding things to your ViewController for Login. We need to make sure we only commit code based on what we say is in our title and description.

Reply:
I added this protocol conformance here because the dependency container will register the LoginVC as a LoginPresenterViewable.

Reply:
I was thinking of making the LoginPresenterViewable methods to be optional, but this will cause me to use the @objc types.Making this methods I won't have to use them now in the LoginVC. What do you think about this approach?

Reply:
no @objc because they dont need to be optional, if you have methods you dont need, that means that protocol does not have a single responsibility.

Reply:
please Remove it as its not being used yet. unless you are using it you dont need it. Please rename this view controller also.